pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -614,6 +614,10 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication, VaultGuard {
         return (_bufferTokenBalances[token].getBalanceRaw(), _bufferTokenBalances[token].getBalanceDerived());
     }
 
+    function isERC4626Initialized(IERC4626 token) external view returns (bool) {
```

Comment:
This function doesn't seem to be used anywhere. Did you mean to add this to the external interface? (In which case it should be in IVaultAdmin.) Or is it left over?

Reply:
It's a left over, I'll remove it!